v8/pretty_breadcrumbs/conf.py.sample

```diff
@@ -0,0 +1,5 @@
+
+# By default, the pretty_crumbs plugin looks for the metadata tag `crumb`.
+# This can be changed with PRETTY_BREADCRUMBS_TAG.
+#PRETTY_BREADCRUMBS_TAG = "nav"
```

Comment:
I honestly wouldn’t bother with configurability.

Reply:
I put the config option in because I use `nav` for historical reasons but it's easy enough to change it to `crumb`. I can drop the config if you don't think it's useful.